File: 백준/Silver/1074. Z/Z.py
n,r,c=map(int,input().split())
ans = 0
for i in range(n-1,-1,-1):
    ans <<= 1
    if r & (1 << i):
        ans += 1
    ans <<= 1
    if c & (1 << i):
        ans += 1
print(ans)

File: 백준/Silver/1074. Z/Z.c
#include <stdio.h>

int main(void) {
	int n, r, c;
    scanf("%d%d%d",&n,&r,&c);
    int ans = 0;
    for(int i=1<<n; i!=0; i>>=1){
        ans <<= 1;
        if (r & i)
            ans += 1;
        ans <<= 1;
        if (c & i)
            ans += 1;
    }
    printf("%d\n",ans);
    return 0;
}